### ISSUE TITLE
Group use declarations

### ISSUE THREAD

**Issue by orlandocharles** [marabesi/php-import-checker#16]
Any chance to implement group use declarations for php 7?

```php
use Foo\Bar\{
    ClassA, 
    ClassB, 
    ClassC
};
```

**Comment by marabesi** [comment]
that's a good one, but would be good to have some pr's as well

**Comment by jackwilsdon** [comment]
I'm thinking it might be worth bringing in something like https://github.com/glayzzle/php-parser to properly parse the code, as opposed to depending on increasingly complex regular expressions which might not catch some valid code or may include invalid code.

**Comment by marabesi** [comment]
@jackwilsdon good point, I think #10 is related to it

**Comment by jackwilsdon** [comment]
Ah I missed that, cheers!

**Comment by marabesi** [comment]
issue potentially closed as version 0.6.0 gives the supported asked, live example:

![image](https://user-images.githubusercontent.com/2129872/144720949-29e7ebad-b1bf-4310-8af6-2547d6c394b1.png)

**Comment by marabesi** [comment]
reopen as the versions released under 0.6.* are all buggy and not working.

refs #303

**Comment by marabesi** [comment]
I released a few improvements on namespaces, would be god to have some feedback on this one. To test it you should have the following settings:

```json
"php.import.highlight": {
   "use_next_version": true
}
```
